modules/imaging_uploader/jsx/columnFormatter.js

```diff
@@ -56,6 +56,11 @@ function formatColumn(column, cell, rowData, rowHeaders) {
   }
 
   if (column === 'Tarchive Info') {
+    if (cell === null || cell === "0") {
```

Comment:
Might as well check for other falsey values too:
```if (!cell || cell === "0") {```

also, do we need to check for the string "0"?